features/works/work_drafts.feature

```diff
@@ -185,3 +185,24 @@ Feature: Work Drafts
       When I follow "Next Chapter"
         And I press "Post Chapter"
       Then I should see "Words:16"
+
+  Scenario: A co-creator, invited or approved, should be able to view chapters in a draft work
+    Given I am logged in as "test_user"
+      And the following activated users exist
+        | login          | email                 |
+        | coauthor       | coauthor@example.org  |
+      And the user "coauthor" allows co-creators
+      And I set up the draft "Unicorns are everywhere"
+      And I fill in "content" with "Help there are unicorns everywhere"
+      And I check "Add co-creators?"
+      And I fill in "pseud_byline" with "coauthor"
+    When I press "Preview"
+      And I press "Save As Draft"
+    When a chapter is set up for "Unicorns are everywhere"
```

Comment:
Do we need to check if the co-creator can look at the second chapter as well? I know this step is here so we can have a multi-chapter work but it feels weird not to do anything else with it.

Reply:
Do we even need this feature test now that we cover co-creator access to draft chapters in the specs?

Reply:
Yeah we're good there. Let's remove this.